packages/apps-v5/test/unit/commands/ps/type.unit.test.js

```diff
@@ -190,16 +190,32 @@ it('errors when user requests larger dynos and feature flag is NOT enabled', fun
   let api = nock('https://api.heroku.com')
     .get('/account/features/frontend-larger-dynos')
     .reply(200, featureFlagPayload())
+    .get('/apps/myapp/formation')
+    .reply(200, [{type: 'web', quantity: 1, size: 'performance-l-ram'}, {type: 'web', quantity: 2, size: 'performance-l-ram'}])
 
   return cmd.run({app: 'myapp', args: ['web=performance-l-ram']})
     .catch(error => expect(error.message).to.eq('No such size as performance-l-ram. Use eco, basic, standard-1x, standard-2x, performance-m, performance-l, private-s, private-m, private-l, shield-s, shield-m, shield-l.'))
     .then(() => api.done())
 })
 
+it('errors when user requests larger dynos and feature flag is NOT enabled when chaning all type sizes', function () {
+  let api = nock('https://api.heroku.com')
+    .get('/account/features/frontend-larger-dynos')
+    .reply(200, featureFlagPayload())
+    .get('/apps/myapp/formation')
+    .reply(200, [{type: 'web', quantity: 1, size: 'performance-2xl'}, {type: 'web', quantity: 2, size: 'performance-2xl'}])
+
+  return cmd.run({app: 'myapp', args: ['performance-2xl']})
+    .catch(error => expect(error.message).to.eq('No such size as performance-2xl. Use eco, basic, standard-1x, standard-2x, performance-m, performance-l, private-s, private-m, private-l, shield-s, shield-m, shield-l.'))
```

Comment:
This seems like the incorrect error message here @zwhitfield3 . When the argument is just a dyno size, it's the format of the argument that is wrong, not that the dyno size does not exist, right?

Reply:
@mars, not in this case. That's the case for `ps:scale` where, if you don't pass a well-formed argument, it just gets ignored. For `ps:type` if you pass the dyno size only, it gets applied to all process types. That's described [here](https://devcenter.heroku.com/articles/dyno-types#change-the-dyno-type-for-all-processes) in Dev Center and it's implemented on the command. That's why we fixed it here. This was a real bug for a use case we didn't account for initially.

Reply:
Ok, thanks for clearing that up @sbosio  😄